Module not found

Hey,

**Thanks for writing this plugin ❤️.**

When I try to use it in my typescript payload app, i get the error `Cannot find module 'payload-cloudinary-plugin' or its corresponding type declarations.` when I try to import it.

If i import it via `import { mediaManagement } from "payload-cloudinary-plugin/src/index";`, then I get an error when i try to run the app ==> `SyntaxError: Cannot use import statement outside a module`

Am I doing something wrong here?

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -13,6 +13,7 @@ jobs:
           node-version: '16.x'
           registry-url: 'https://registry.npmjs.org'
       - run: yarn
+      - run: yarn build
       - run: yarn publish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}

--- a/.gitignore
+++ b/.gitignore
@@ -103,3 +103,4 @@ dist
 # TernJS port file
 .tern-port
 .DS_Store
+test_usage

--- a/package.json
+++ b/package.json
@@ -6,6 +6,9 @@
   "repository": "https://github.com/finkinfridom/payload-auth0-plugin.git",
   "author": "a.maghini@gmail.com",
   "license": "MIT",
+  "files": [
+    "dist"
+  ],
   "scripts": {
     "dev": "tsc --watch",
     "build": "tsc",

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,27 +1,19 @@
 {
-    "compilerOptions": {
-        "target": "es5",
-        "lib": [
-            "dom",
-            "dom.iterable",
-            "esnext"
-        ],
-        "allowJs": true,
-        "strict": false,
-        "esModuleInterop": true,
-        "skipLibCheck": true,
-        "outDir": "./dist",
-        "rootDir": "./src",
-        "jsx": "react"
-    },
-    "include": [
-        "src"
-    ],
-    "exclude": [
-        "node_modules",
-        "dist"
-    ],
-    "ts-node": {
-        "transpileOnly": true
-    }
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "strict": false,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "outDir": "./dist",
+    "rootDir": "./src",
+    "jsx": "react",
+    "declaration": true
+  },
+  "include": ["src"],
+  "exclude": ["node_modules", "dist"],
+  "ts-node": {
+    "transpileOnly": true
+  }
 }

